### PR TITLE
Add set of chrono types guarded behind the chrono feature.

### LINKED ISSUE #46: Add support for Chrono

### PR DESCRIPTION
**Pull request by xkikeg**
Closes #46

### CODE REVIEW

**Comment by xkikeg** [comment]
Give me any feedback, basically I'm not quite familiar with Rust development and how the Cargo treats the dependencies yet.

**Comment by fujiapple852** [comment]
Thanks for the PR @xkikeg !

Overall LGTM, I've rebased and tweaked it a little (see review comments above).

**Comment by xkikeg** [comment]
Thanks! LGTM.

File: bounded-static/src/lib.rs [hunk 1]
```diff
@@ -235,18 +250,18 @@ impl IntoBoundedStatic for &'static str {
     }
 }
 
-/// No-op [`ToBoundedStatic`] and [`IntoBoundedStatic`] impls for primitive types.
-macro_rules! make_primitive_impl {
```
Comment (xkikeg) [inline bounded-static/src/lib.rs:239]: Thanks for the splitting the commit, yup that was lazy

**Review by fujiapple852 — APPROVED** [review]

File: bounded-static/Cargo.toml [hunk 2]
```diff
@@ -21,17 +21,21 @@ alloc = []
...
 
 # Enable impls of [To|Into]BoundedStatic for other types in std.
-std = [ "alloc", "ahash?/std" ]
+std = [ "alloc", "ahash?/std", "chrono?/std" ]
 
 # Enable the ToStatic custom derive macro.
 derive = [ "bounded-static-derive" ]
 
+# Enable the clock feature for chrono.
+chrono-clock = [ "chrono", "chrono/clock" ]
```
[the remark this answers is not in the document]
Comment (xkikeg) [reply bounded-static/Cargo.toml:30]: Thanks for adding this! I hope there's no other types behind the guard

**Pull request merged**